infra/modules/service/outputs.tf

```diff
@@ -24,3 +24,23 @@ output "migrator_role_arn" {
   description = "ARN for role to use for migration"
   value       = length(aws_iam_role.migrator_task) > 0 ? aws_iam_role.migrator_task[0].arn : null
 }
+
+output "cluster_arn" {
+  value = aws_ecs_cluster.cluster.arn
+}
+
+output "task_definition_arn" {
+  value = aws_ecs_task_definition.app.arn
+}
+
+output "task_role_arn" {
+  value = aws_iam_role.task_executor.arn
+}
+
+output "app_security_group_id" {
+  value = aws_security_group.app.id
+}
+
+output "service_logs_arn" {
+  value = aws_cloudwatch_log_group.service_logs.arn
+}
```

Comment:
All of these outputs are a bit of a code smell. But. When you put the state machine inside of this module, you have to wrap it in a boolean so that the frontend doesn't try to deploy it.